lenses-alerts-scala/src/test/scala/io/lenses/alerting/plugin/scalaapi/AlertingServiceSpec.scala

```diff
@@ -32,6 +37,9 @@ class AlertingServiceSpec extends AlertingSpecBase {
     "fail on underlying service publish raises exception" in new TestContext {
       val service = new AlertingService(new JAlertingService {
         override def publish(alert: JAlert): Try[JAlert] = throw anException
+        override def name(): String = ""
+        override def description(): String = ""
+        override def displayedInformation(): java.util.Map[String, String] = Collections.emptyMap()
```

Comment:
creating dummy AlertingService defining all those methods once may be nice, but not necessary